src/main/java/ru/kids/bot/model/CustomerRepository.java

```diff
@@ -0,0 +1,6 @@
+package ru.kids.bot.model;
+
+import org.springframework.data.repository.CrudRepository;
+
+public interface CustomerRepository extends CrudRepository<Customer,Long> {
```

Comment:
Забыл указать анатацию @Repository

Reply:
ошибка осталась та же самая java.lang.NullPointerException: Cannot invoke "ru.kids.bot.model.CustomerRepository.findById(Object)" because "this.customerRepository" is null